heroic-core/src/main/java/com/spotify/heroic/http/tracing/OpenCensusApplicationEventListener.java

```diff
@@ -25,6 +25,7 @@
 import static java.text.MessageFormat.format;
 
 import com.spotify.heroic.common.ServiceInfo;
+import com.spotify.heroic.tracing.EnvironmentMetadata;
```

Comment:
Unused import - probably why the build is failing

Reply:
this is from refactoring this module - it's used below `private static final EnvironmentMetadata environmentMetadata = EnvironmentMetadata.create();`